app/main.py

```diff
@@ -1,3 +1,55 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int) -> Distance:
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+            return self
+        self.km = self.km + other
+        return self
+
+    def __mul__(self, other: Distance | int) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: Distance | int) -> Distance:
```

Comment:
Guide about __truediv__ told, that:

__truediv__(self, other)
Implements true division. Note that this only works when from __future__ import division is in effect.

I don't know is it nessesary in modern version of Python, so just a noticed and check this.